### PR TITLE
Add comments form

### PR DESCRIPTION
**Pull request by ankitamitra**
Add comments form at the bottom. Doesn't store comments yet. 

From part 4 walkthrough spec: 
- add a form element to your portfolio page. It's up to you what input elements are in the form, but at a minimum you probably want a text input for the user to add a comment.
- add a doPost() function to your DataServlet.java file that takes the data from the request to populate your comments data structure. An ArrayList of String values is fine.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -48,4 +49,35 @@ private String convertToJson(ArrayList<String> messages) {
...
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String name = getParameter(request, "name", "");
+    String email = getParameter(request, "email", "");
+    String subject = getParameter(request, "subject", "");
+    String comments = getParameter(request, "comments", "");
+    
+    messages = new ArrayList<String>();
+    messages.add(0, comments);
```
Comment (cjwert-g) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:62]: Why use add(index, value) rather than just add(value)?

Comment (ankitamitra) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Oops - was doing some experiments. Fixed + deployed :)

**Comment by cjwert-g** [comment]
Nice work! :)

**Comment by cjwert-g** [comment]
Nice work!

**Pull request merged**